### PR TITLE
Generating static scene using stability.ai

### PR DESCRIPTION
**Pull request by bartoszrud**
Addressing #9

### CODE REVIEW

File: src/videopython/utils/stability_generation.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import io
+import os
+import warnings
+import random
+
+import numpy as np
+import stability_sdk.interfaces.gooseai.generation.generation_pb2 as generation
+from PIL import Image
+from videopython.project_config import LocationConfig
+from stability_sdk import client
```
Comment (BartWojtowicz) [inline src/videopython/utils/stability_generation.py:10]: Can we do isort? 🙏🏻

File: src/videopython/utils/stability_generation.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    steps: int = 30,
+    cfg_scale: float = 8.0,
+    engine: str = "stable-diffusion-xl-1024-v1-0",
+    verbose: bool = True,
+    seed: int = 1,
+) -> tuple[np.ndarray, str]:
+    """Generates image from prompt using the stability.ai API."""
+
+    stability_api = client.StabilityInference(
+        key=os.environ["STABILITY_KEY"],
```
Comment (BartWojtowicz) [inline src/videopython/utils/stability_generation.py:27]: Can we move it to `project_config.py` to a new class for storing API keys?

File: src/videopython/utils/stability_generation.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        verbose=verbose,
+        engine=engine,  # Set the engine to use for generation.
+        # Check out the following link for a list of available engines: https://platform.stability.ai/docs/features/api-parameters#engine
+    )
+
+    # Set up our initial generation parameters.
+    answers = stability_api.generate(
+        prompt="expansive landscape rolling greens with gargantuan yggdrasil, intricate world-spanning roots towering under a blue alien sky, masterful, ghibli",
```
Comment (BartWojtowicz) [inline src/videopython/utils/stability_generation.py:35]: Shouldn't we change this prompt?

File: src/videopython/utils/stability_generation.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        )
+        save_dir = LocationConfig.generated_images_dir / filename
+
+    for resp in answers:
+        for artifact in resp.artifacts:
+            if artifact.finish_reason == generation.FILTER:
+                warnings.warn(
+                    "Your request activated the API's safety filters and could not be processed."
+                    "Please modify the prompt and try again."
+                )
```
Comment (BartWojtowicz) [inline src/videopython/utils/stability_generation.py:63]: Can we somehow turn this off? 😎

File: requirements.txt [hunk 5]
```diff
@@ -5,4 +5,5 @@ opencv-python==4.7.0.68
 pytest==7.4.0
 transformers==4.35.0
 diffusers==0.21.4
-torch==2.1.0
\ No newline at end of file
+torch==2.1.0
+stability-sdk==0.8.4
```
Comment (BartWojtowicz) [inline requirements.txt:9]: Can we add this to `pyproject.toml` as well?

File: src/videopython/utils/stability_generation.py (hunk 4 above)
Comment (bartoszrud) [reply src/videopython/utils/stability_generation.py:63]: I don't think so :/

**Pull request merged**